Mathlib/MeasureTheory/Group/Arithmetic.lean

```diff
@@ -524,6 +523,11 @@ instance (priority := 100) measurableDiv₂_of_mul_inv (G : Type*) [MeasurableSp
 #align has_measurable_div₂_of_mul_inv measurableDiv₂_of_mul_inv
 #align has_measurable_div₂_of_add_neg measurableDiv₂_of_add_neg
 
+-- See note [lower instance priority]
+instance (priority := 100) MeasurableDiv.toMeasurableInv [MeasurableSpace α] [Group α]
+    [MeasurableDiv α] : MeasurableInv α where
+  measurable_inv := by simpa using measurable_const_div (1 : α)
+
```

Comment:
These look unrelated to the current PR; perhaps merging master will make them go away?

Reply:
Yes, thanks. Now only model theory files are affected.